src/identities/identities.service.spec.ts

```diff
@@ -290,4 +291,21 @@ describe('IdentitiesService', () => {
       expect(result).toEqual(mockAssets);
     });
   });
+
+  describe('getPendingDistributions', () => {
+    it('should return the Dividend Distributions associated with an Asset that have not been claimed', async () => {
+      const mockDistributions = [new MockDistributionWithDetails()];
+
+      const mockIdentity = new MockIdentity();
```

Comment:
```suggestion
      const mockIdentity = createMock<Identity>()
```
(nit) the auto mock shouldn't need an `as any` cast so its a bit cleaner imo.